src/main/scala/cognite/spark/v1/DefaultSource.scala

```diff
@@ -406,8 +406,8 @@ object DefaultSource {
     val collectMetrics = toBoolean(parameters, "collectMetrics")
     val collectTestMetrics = toBoolean(parameters, "collectTestMetrics")
 
-    val enableSinglePartitionDeleteAssetHierarchy =
-      toBoolean(parameters, "enableSinglePartitionDeleteHierarchy", defaultValue = false)
+    val userSinglePartition =
+      toBoolean(parameters, "useSinglePartition", defaultValue = false)
```

Comment:
```suggestion
      toBoolean(parameters, "useSinglePartitionForAssetHierarchyDeletes", defaultValue = true)
```
not sure about the name...